.github/workflows/functional-test.yml

```diff
@@ -2,7 +2,7 @@ name: Functional Tests
 on:
   workflow_dispatch:
   pull_request:
-    branches: [ main ]
+    branches: [ develop ]
```

Comment:
I think we can leave the MFTF tests to only be executed on a release PR, instead of having them run on every PR

Reply:
Thank you for you review @Morerice, I converted against develop for only testing. I will revert back after completing the PR.